.github/workflows/daphneci.yml

```
@@ -39,8 +39,35 @@ jobs:
     steps:
       - name: Checking out
         uses: actions/checkout@v3
+
+      - name: Generate HPKE Signing Key
+        id: hpke_signing_key
```

For consistency with `DAP_SIGNING_KEY`. If you prefer HPKE signing key, that's fine, let's just be consistent.

Here and below.
```suggestion
      - name: Generate DAP Signing Key
        id: dap_signing_key
```

---

changed all `DAP_SIGNING_KEY` to `HPKE_SIGNING_KEY`